Userland/Libraries/LibJS/Runtime/Intl/DateTimeFormat.h

```diff
@@ -162,10 +161,7 @@ enum class OptionDefaults {
 // Table 8: Record returned by ToLocalTime, https://tc39.es/ecma402/#table-datetimeformat-tolocaltime-record
 // Note: [[InDST]] is not included here - it is handled by LibUnicode / LibTimeZone.
 struct LocalTime {
-    AK::Time time_since_epoch() const
-    {
-        return AK::Time::from_timestamp(year, month + 1, day + 1, hour, minute, second, millisecond);
-    }
+    AK::Time time_since_epoch() const;
```

Comment:
Defining this out of line is fine, but does not make `Time` forward declarable here.